examples/011_rst_workflow.py

```diff
@@ -0,0 +1,89 @@
+"""
+.. _rst_workflow_example:
+
+Failure Analysis of an MAPDL (RST) Model
+----------------------------------------
+
+The post-processing of a MAPDL (RST) model with layered elements which was not
+preprocessed by ACP is shown by this example. The difference between the RST
+only and ACP based workflow is that `composite` of :class:`.ContinuousFiberCompositesFiles`
+is empty and so the section data are automatically loaded from the RST file.
+
+The engineering data file (XML or ENGD) with the material properties is needed anyway.
+Otherwise, the material properties cannot be mapped. It is recommended
+to create it before solving the model. The engineering data file can be either generated with Ansys Workbench
+or ACP (Ansys Composite PrePost) standalone. Important: The material UUIDs in the engineering data file
+must be identical to the UUIDs in the Mechanical APDL (RST file).
+The material UUID can be set in Mechanical APDL with the command ``MP,UVID,<material index>,<value>``.
+
+This workflow is supported since version 2024 R2 (DPF Server version 8.0). A few advanced features are
+not supported with the RST onl workflow. Refer to Section :ref:`limitations` for details.
```

Comment:
```suggestion
This workflow is supported in 2024 R2 (DPF Server version 8.0) and later. A few advanced features are
not supported with the RST onl workflow. For more information, see :ref:`limitations`.
```
Not sure about "RST only workflow". Should it be RST-only workflow?